config/config.go

```diff
@@ -263,6 +263,34 @@ func (c *toEnable) LogConfig(logger *logrus.Entry) {
 	logger.Info("enabled")
 }
 
+type SourceLoadingConfig struct {
+	DownloaderConfig DownloaderConfig `yaml:",inline"`
+
+	Concurrency        uint `yaml:"concurrency" default:"4"`
+	MaxErrorsPerSource int  `yaml:"maxErrorsPerSource" default:"5"`
+}
+
+func (c *SourceLoadingConfig) LogConfig(logger *logrus.Entry) {
+	logger.Infof("concurrency = %d", c.Concurrency)
```

Comment:
Should it not also log all parameters from "DownloaderConfig"?

Reply:
Was a bit afraid that'd be a lot of output.  
I added those, but also changed some to `Debugf`. Let me know if you think I should adjust the levels I chose.